File: web/app/model/Polygon.js
Ext.define('Traccar.model.Polygon', {
    extend: 'Ext.data.Model',
    identifier: 'negative',
    fields: [{
        name: 'id',
        type: 'int'
    }, {
        name: 'type',
        type: 'string'
    }, {
        name: 'name',
        type: 'string'
    }, {
        name: 'coordinates'
    }]
});

File: src/main/java/org/traccar/model/Polygon.java
package org.traccar.model;

import java.util.List;

/**
 * Created by niko on 12/19/15.
 */
public class Polygon{
    private long id;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    private String type;

    private String name;

    private List<Point> coordinates;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Point> getCoordinates() {
        return coordinates;
    }

    public void setCoordinates(List<Point> coordinates) {
        this.coordinates = coordinates;
    }
}
